rpc/wrtc_signaling_answerer.go

```diff
@@ -331,9 +331,13 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		exchangeCtx, exchangeCancel = context.WithTimeout(ans.closeCtx, getDefaultOfferDeadline())
 	}
 
-	errCh := make(chan interface{})
+	errCh := make(chan error)
 	defer exchangeCancel()
-	sendErr := func(err interface{}) {
+	sendErr := func(err error) {
+		if isEOF(err) {
+			ans.logger.Warnf("answerer swallowing err %v", err)
+			return
+		}
```

Comment:
Can we put another warn log line after this block showing the error with `%v` and `%T` (type)? If an error besides `EOF` slips into here, we want to know what it was so we can decide if we want to add it to our "allowlist" (currently we're just allowing `EOF`).